tool/build_release.sh

```diff
@@ -4,7 +4,27 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-set -ex #echo on
+# Use the Flutter SDK from flutter-sdk/.
+FLUTTER_DIR="`pwd`/flutter-sdk"
+PATH="$FLUTTER_DIR/bin":$PATH
+
+REQUIRED_FLUTTER_VERSION=$(<"flutter-version.txt")
+
+flutter --version
+ACTUAL_FLUTTER_VERSION=$(<"$FLUTTER_DIR/version")
+
+# Check that the 'actual' and 'required' SDK versions agree.
+if [[ "$REQUIRED_FLUTTER_VERSION" != "$ACTUAL_FLUTTER_VERSION" ]]; then
+  echo ""
+  echo "flutter-version.txt != flutter-sdk/version"
+  echo "  $REQUIRED_FLUTTER_VERSION != $ACTUAL_FLUTTER_VERSION"
+  echo ""
+  echo "To switch versions, run './tool/update_flutter_sdk.sh'."
```

Comment:
instead of failing if we are on the wrong version, can update_flutter_sdk.sh be ran automatically as part of this build release script?

Reply:
We could, but I think it's useful for developers to be aware that they need to be building / running with the same SDK as what we ship with. I.e., prompting them to run `tool/update_flutter_sdk.sh` is probably a good reminder about how to keep your local version synced.